addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/internal/command/PostLogin.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.internal.command;
+
+import static org.openhab.binding.solaredge.SolarEdgeBindingConstants.*;
+
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Result;
+import org.eclipse.jetty.client.util.FormContentProvider;
+import org.eclipse.jetty.http.HttpMethod;
+import org.eclipse.jetty.util.Fields;
+import org.openhab.binding.solaredge.handler.SolarEdgeHandler;
+import org.openhab.binding.solaredge.internal.callback.AbstractCommandCallback;
+import org.openhab.binding.solaredge.internal.connector.StatusUpdateListener;
+
+/**
+ * implements the login to the webinterface
+ *
+ * @author afriese
```

Comment:
Author should be real name and added comment. Like: `- Initial contribution`

Reply:
fixed it